packages/react-router/dom-export.tsx

```diff
@@ -0,0 +1,3 @@
+export type { RouterProviderProps } from "./lib/dom-export/dom-router-provider";
+export { RouterProvider } from "./lib/dom-export/dom-router-provider";
+export { HydratedRouter } from "./lib/dom-export/hydrated-router";
```

Comment:
Naming of this file may be temporary - went with `dom-export` to avoid ambiguity with the existing `dom/` folder containing the old `react-router-dom` implementation - but open to suggestions